xchange-okex/src/main/java/org/knowm/xchange/okex/v5/OkexAdapters.java

```diff
@@ -81,32 +89,106 @@ public static OkexAmendOrderRequest adaptAmendOrder(LimitOrder order) {
   }
 
   public static OkexOrderRequest adaptOrder(LimitOrder order) {
+    String positionEffect = null;
+    switch (order.getType()) {
+      case BID:
+        positionEffect = "long";
+        break;
+      case ASK:
+        positionEffect = "short";
+        break;
+      case EXIT_ASK:
+        positionEffect = "short";
+        break;
+      case EXIT_BID:
+        positionEffect = "long";
+        break;
+    }
```

Comment:
please extract this to a method `adaptOrderTypeToPositionEffect`

Reply:
done